pool-reference/templates/chia-pool.service.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Chia Pool Service for {{ user }}
+
+[Service]
+Type=simple
```

Comment:
Asking because I dunno. Should this be forking, or is this just one process starting up?

Reply:
This one is just a single process starting and running, pretty sure